app/models/listing.rb

```diff
@@ -52,6 +52,18 @@ def view_price
     tickets.first.format_price
   end
 
+  def listing_category
+    event.category.name
+  end
+
+  def seats
+    tickets.map do |ticket|
+      if ticket.status == "active"
+        ticket.seat
+      end
+    end
```

Comment:
I think we can do this with an Active Record call: tickets.where(status: "active).pluck(:seat) (?) something like that...